snakesist/exist_client.py

```diff
@@ -26,7 +27,7 @@
 DEFAULT_PARSER = etree.XMLParser(recover=True)
 
 
-class Resource:
+class Resource(ABC):
```

Comment:
`delete` and `update_push` should be declared as `abc.abstractmethod` in this class.

Reply:
:+1: done